.github/workflows/ci-workflow.yml

```diff
@@ -29,7 +29,7 @@ jobs:
             az account show
             az storage -h
           EXPECTED_TO: pass
-        run: ts-node test/main.test.ts
+        run: ts-node src/test/main.test.ts
```

Comment:
Why have you moved test into src?

Reply:
Because of an issue while compiling the build.
https://github.com/t-dedah/cli/runs/4124823760?check_suite_focus=true

Reply:
Will the test files part of build artifact? There must be other option to skip test folder in artifacts.

Reply:
we can explicitly set a version of ncc to a lower version like in arm-deploy and resolve this issue.

Reply:
How it was working earlier? What has changed that we needed to move test to src ?

Reply:
We never specified the ncc version so as long as old verson worked it was working fine but now they are on version which doesnt allow it. 
This is an assumption based on package.json for both actions.